src/models/searchservice/response/FeaturedSnippetDirectAnswer.ts

```diff
@@ -18,7 +20,7 @@ export interface FeaturedSnippetDirectAnswer extends DirectAnswer {
   /** {@inheritDoc DirectAnswer.verticalKey} */
   verticalKey: string,
   /** {@inheritDoc DirectAnswer.fieldType} */
-  fieldType: string,
+  fieldType: EnumOrLiteral<FieldType>,
```

Comment:
why does this field need to be specified again on `FeaturedSnippetDirectAnswer` when it's already in the extended `DirectAnswer` model? it doesn't look like we're changing the type either

Reply:
looks like several fields in `FeaturedSnippetDirectAnswer` and `FieldValueDirectAnswer` are redefined without changes

Reply:
I believe someone mentioned in previous prs that our interfaces duplicate fields so it's more readable in the docs, although I'm not oppose to delete it now.

Reply:
I also don't mind deleting the duplication. On one hand it's easier to read the code since you don't have to go up inheritance trees. On the other hand there's more code and it's more annoying to refactor stuff and write new code. I don't feel super strongly either way, readability of docs is nice but shouldn't have a large impact on how we write code. If we want better docs we can improve the way they are generated.

Reply:
asked tom, he's fine leaving it as is for now. If we decide to refactor later on, it wouldn't be a breaking change.